src/org/usfirst/frc/team694/robot/commands/auton/routines/DoubleCubeSwitchAutonCommand.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team694.robot.commands.auton.routines;
+
+import org.usfirst.frc.team694.robot.commands.LiftMoveToBottomCommand;
+import org.usfirst.frc.team694.robot.commands.LiftMoveToHeightCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorAcquireCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorCloseCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorDeacquireCommand;
+import org.usfirst.frc.team694.robot.commands.QuisitorOpenCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DriveStraightRampDownOnlyCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DriveStraightWithRampingCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainDriveCurveCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainMoveInchesEncoderCommand;
+import org.usfirst.frc.team694.robot.commands.auton.DrivetrainRotateAbsoluteDegreesPIDCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class DoubleCubeSwitchAutonCommand extends CommandGroup {
+    private static final double DISTANCE_TO_CUBE = 100;
+    private static final double DISTANCE_TO_SWITCH = 100;
+
+    public DoubleCubeSwitchAutonCommand() {
+        DrivetrainDriveCurveCommand curveToCube = new DrivetrainDriveCurveCommand(DISTANCE_TO_CUBE);
+        curveToCube.addSpeedChange(0, -0.6);
+        curveToCube.addTurn(40, 90);
+
+        DrivetrainDriveCurveCommand curveToSwitch = new DrivetrainDriveCurveCommand(DISTANCE_TO_SWITCH);
+        curveToSwitch.addSpeedChange(0, 0.6);
+        curveToSwitch.addTurn(40, 0);
+
+        addSequential(new SingleCubeSwitchAutonCommand());
+        addSequential(curveToCube, 5); //TODO: Is this the right amt of secs?
+        addParallel(new LiftMoveToBottomCommand());
+        addSequential(new DrivetrainRotateAbsoluteDegreesPIDCommand(0));
+        addSequential(new QuisitorOpenCommand());
+        addSequential(new QuisitorAcquireCommand(),2);
```

Comment:
Needs to be parallel. Otherwise this will wait 2 seconds and stop acquiring before moving forward